benchexec/tablegenerator/react-table/.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
There is already an `eslintConfig` in `package.json`. We should not have two places where this is configured, so I suggest to move this to `package.json`.

Reply:
Will do